src/Rules/StarterWeb/AI/IndividualAuth/appsettings.json

```diff
@@ -2,11 +2,7 @@
   "ApplicationInsights": {
     "InstrumentationKey": ""
   },
-  "Data": {
-    "DefaultConnection": {
-      "ConnectionString": "Server=(localdb)\\mssqllocaldb;Database=aspnet-$safeprojectname$-$guid2$;Trusted_Connection=True;MultipleActiveResultSets=true"
-    }
-  },
+  "DefaultConnection": "Server=(localdb)\\mssqllocaldb;Database=aspnet-$safeprojectname$-$guid2$;Trusted_Connection=True;MultipleActiveResultSets=true",
```

Comment:
"DefaultConnection" is the connection string name developers have come to expect from the FNP experience. We also added UI to the SQL server/db-creation process in SDK 2.8.2 to re-surface the connection string per customer requests to make the experience more like the pre-2.8.1 experience. Users *expect* to see the name of their DB connection string as DefaultConnection